app/assets/javascripts/discourse/app/templates/components/composer-editor.hbs

```diff
@@ -20,6 +20,13 @@
   disabled=disableTextarea
   outletArgs=(hash composer=composer editorType="composer")}}
 
+{{#if experimentalComposerUploads}}
+  <br>
```

Comment:
why a br and not padding/margin ?

Reply:
It's just a message for us to know this is enabled when we turn it on internally, this will be removed later so no need for any sort of customization, the lazy way is fine here IMO